From 545e4197258d8243ede63314faaa14fa260d450e Mon Sep 17 00:00:00 2001
From: dummy <dummy@example.com>
Date: Mon, 26 May 2014 01:33:32 +0000
Subject: [PATCH] avoid building warp executable

fails to link for some reason

/home/builder/.ghc/android-14/arm-linux-androideabi-4.8/bin/../lib/gcc/arm-linux-androideabi/4.8/../../../../arm-linux-androideabi/bin/ld: fatal error: /home/builder/.ghc/android-14/arm-linux-androideabi-4.8/.cabal/lib/arm-linux-android-ghc-7.8.2/system-filepath-0.4.11/libHSsystem-filepath-0.4.11.a(Path.o): unsupported ELF machine number 0
---
 wai-app-static.cabal |   11 -----------
 1 file changed, 11 deletions(-)

diff --git a/wai-app-static.cabal b/wai-app-static.cabal
index e5d11a6..d4fd0d1 100644
--- a/wai-app-static.cabal
+++ b/wai-app-static.cabal
@@ -67,17 +67,6 @@ library
     if flag(print)
       cpp-options:  -DPRINT
 
-Executable warp
-  Main-is:        warp-static.hs
-  hs-source-dirs: app
-  Build-depends: base            >= 4                  && < 5
-               , wai-app-static  >= 2.0                && < 2.1
-               , directory       >= 1.0
-               , containers      >= 0.2
-               , bytestring      >= 0.9.1.4
-               , text            >= 0.7
-               , mime-types      >= 0.1                && < 0.2
-
 test-suite runtests
     hs-source-dirs: test
     main-is: ../tests.hs
-- 
1.7.10.4

